bitnami/prometheus-operator/values-production.yaml

```diff
@@ -642,6 +642,18 @@ prometheus:
     ##
     # key:
 
+  ## Enable additional Prometheus alert relabel configs that are managed externally to this chart
+  ## Note that the prometheus will fail to provision if the correct secret does not exist.
+  ## ^^ Maybe??
```

Comment:
@juan131 Do you know if this is actually the case if the secret is not entered correctly? It behaves the same as `additionalScrapeConfigsExternal` ?